src/main/java/seedu/commands/BudgetCommand.java

```diff
@@ -120,13 +133,28 @@ private void addBudget(SpendingList spendings, double budget, String timeRange)
         }
     }
 
+    private static void checkUserBudgetEntered(double smallerBudget, double biggerBudget, String smallerBudgetType,
+            String biggerBudgetType) {
+        if (biggerBudget <= MINIMUM_AMOUNT_ENTERED) {
+            throw new WiagiInvalidInputException(INVALID_AMOUNT + ENTER_BUDGET_MESSAGE);
+        } else if (biggerBudget > MAX_LIST_TOTAL_AMOUNT) {
```

Comment:
I think there is no need for `else if` can be just `if`

Reply:
recitified